### PR TITLE
Fix spelling of PORTRAIT.

### PR DESCRIPTION
**Pull request by tpdi**
Correct name of constant.

### CODE REVIEW

File: demo/src/main/scala/demo/components/materialui/MuiDatePickerDemoX.scala [hunk 1]
```diff
@@ -7,7 +7,7 @@ import chandu0101.scalajs.react.components.materialui.{MuiDatePicker, MuiDatePic
 import japgolly.scalajs.react.ReactComponentB
 import japgolly.scalajs.react.vdom.prefix_<^._
 
-object MuiDatePickerDemo {
+object MuiDatePickerDemoX {
```
Comment (chandu0101) [inline demo/src/main/scala/demo/components/materialui/MuiDatePickerDemoX.scala]: any specific reason for this change?

Comment (tpdi) [inline demo/src/main/scala/demo/components/materialui/MuiDatePickerDemoX.scala]: Yes, It's a mistake.

Comment (chandu0101) [inline demo/src/main/scala/demo/components/materialui/MuiDatePickerDemoX.scala]: can you please remove that X then i am happy to merge :)

**Comment by tpdi** [comment]
This replaces the previous pull request.

**Comment by chandu0101** [comment]
cool ok , i am closing this then :)

**Comment by tpdi** [comment]
You don't want to merge it?

**Comment by chandu0101** [comment]
its already fixed in combined example PR  :)

**Comment by tpdi** [comment]
That is true.

**Pull request closed without merging**